### DISCUSSION TITLE
Proxy support with a subpath

### DISCUSSION THREAD

**Discussion by GaPhi** [teslamate-org/teslamate#678] [Ideas]
**What would you like to be added**:
Support of a subpath URL to access TeslaMate, like in Grafana for example.
      - GF_SERVER_DOMAIN=`www.myorg.com`
      - GF_SERVER_ROOT_URL=%(protocol)s://%(domain)s:%(http_port)s`/grafana/`
      - GF_SERVER_SERVE_FROM_SUB_PATH=`true`
And in Apache2 :
	ProxyPass        "/grafana"   "http://`localip`:3000/grafana"
	ProxyPassReverse "/grafana"   "http://`localip`:3000/grafana"

**Why is this needed**:
I would like to use a single public hostname but access several services with a subpath :
- /grafana
- /teslamate
- ...

**Additional context**:
Already working with Grafana and other services but did not succeed with TeslaMate (no dedicated setting found, and Apache2 config file is not sufficient).
	ProxyPass        "/teslamate" "http://`localip`:4000/"
	ProxyPassReverse "/teslamate" "http://`localip`:4000/"

**Comment by SemoTech** [comment]
Hi @GaPhi looks like @tacotran had posted what you need, while trying to help me.  See [HERE](https://github.com/adriankumpf/teslamate/issues/643#issuecomment-622245924).

Does that help?

**Comment by GaPhi** [comment]
Hi, thank you for the link.

I think it is just part of the solution as the teslamate URI shall still be a root base (/index.html is teslamate, I would like to have /teslamate/index.html to be the base of all the URI).

Or did I miss something ?

**Comment by SemoTech** [comment]
Welcome @GaPhi.  I recommend reaching out to @tacotran (tagged here) in case he may be able to help you tweak his solution...  If you do get it to work, kindly share a detailed tutorial, so others can benefit as well.

**Comment by brackendawson** [comment]
I think this is a duplicate of #494
